Example/StripeConnectExample/StripeConnectExample/API/API.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  API.swift
+//  StripeConnect Example
+//
+//  Created by Chris Mays on 8/23/24.
+//
+
+import Foundation
+
+struct API {
+    static func appInfo(baseURL: String = AppSettings.shared.selectedServerBaseURL) async -> Result<AppInfo, APIError> {
+        await apiRequest(path: "app_info", baseURL: baseURL)
+    }
+    
+    static func accountSession(merchantId: String? = nil, baseURL: String = AppSettings.shared.selectedServerBaseURL) async -> Result<AccountSessionResponse, APIError> {
+        await apiRequest(path: "account_session", 
+                         method: "POST",
+                         headers: merchantId.map { ["account": $0] } ?? [:],
+                         baseURL: baseURL)
+    }
+    
+    static func apiRequest<Response: Codable>(path: String,
+                                              method: String = "GET",
+                                              headers: [String: String] = [:],
+                                              baseURL: String = AppSettings.shared.selectedServerBaseURL) async -> Result<Response, APIError> {
+        guard let baseUrl = URL(string: baseURL) else {
+            return .failure(.invalidURL)
+        }
+        let url = baseUrl.appending(path: path)
+        var request = URLRequest(url: url)
+        request.httpMethod = method
+        request.allHTTPHeaderFields = headers
+        do {
+            let (data, response)  = try await URLSession.shared.data(for: request)
+            guard let httpResponse = response as? HTTPURLResponse,
+                  httpResponse.statusCode == 200 else {
+                let decoder = JSONDecoder()
+                decoder.keyDecodingStrategy = .convertFromSnakeCase
+                do {
+                    return .failure(.responseError(response: try decoder.decode(APIErrorResponse.self, from: data)))
+                } catch {
+                    return .failure(.failedToParse(error: error))
+                }
+            }
+            let decoder = JSONDecoder()
+            decoder.keyDecodingStrategy = .convertFromSnakeCase
```

Comment:
Could we reuse the same decoder instead of instantiating two of them? It looks like they use the same configuration